internal/cmd/schema-registry/command_config_delete.go

```diff
@@ -81,6 +82,7 @@ func (c *command) configDelete(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	output.Printf("Deleted %s.\n", resource.SchemaRegistryConfig)
```

Comment:
We should match the style of other delete commands and say "Deleted Schema Registry configuration XXX". I think there's a global constant for this.

Reply:
yeah but it's easier to see the deleted configs as a table.

Reply:
Aren't we only deleting one configuration here? Why would we need a table?

Reply:
configuration includes a --compatibility-group field too

Reply:
```
mhe@C02DV0KVMD6T cli % confluent sr config delete --force                                                                         
Deleted Schema Registry configuration.
+---------------------+---------------------+
| Compatibility Level | BACKWARD            |
| Compatibility Group | application.version |
+---------------------+---------------------+
```